README.md

```diff
@@ -14,12 +14,33 @@ Also, check out the latest version deployed in [Storybook](https://chameleon.ebu
 
 ## Installation
 
+### Install the library in your project
```

Comment:
Probably wrong order, first I'd have project installation instruction, then I'd have this as part of the usage section? Ask for feedback in Slack

Reply:
I changed it